backend/src/main/java/jonaszeihe/ninjin/model/Course.java

```diff
@@ -13,7 +12,7 @@
 @Builder
 @Document(collection = "courses")
 public class Course {
-    @Id
+
```

Comment:
i would add the id field that mongo db create implicitly for you
```suggestion
@Id
private String id;
```